### DISCUSSION TITLE
Imaginary Number

### DISCUSSION THREAD

**Discussion by lt25106** [ManimCommunity/manim#4010] [Suggestions and Proposals]
Kind of like `DecimalNumber()`, but with imaginary numbers.
```
ImaginaryNumber(real,imaginary)=DecimalNumber(real)+MathTex("+i")+DecimalNumber(imaginary)
```

**Comment by chopan050** [comment]
Hello, and thanks for the proposal! I have to mention two things:
- If it has both a real and an imaginary part, it should be called `ComplexNumber`.
- Yesterday, I was adding typehints for `DecimalNumber` and discovered that it actually also accepts complex numbers. Try, for example, `DecimalNumber(1 + 3j)`.

**Reply by lt25106** [reply]
Oh its `j`?

**Reply by behackl** [reply]
Yes: https://docs.python.org/3/library/cmath.html
